lib/config/presets/internal/replacements.ts

```diff
@@ -181,6 +182,17 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'eslint-plugin-node-to-maintenance-fork': {
+    description: 'Maintenance fork of `eslint-plugin-node`',
+    packageRules: [
+      {
+        matchDatasources: ['npm'],
+        matchPackageNames: ['eslint-plugin-node'],
+        replacementName: 'eslint-plugin-n',
+        replacementVersion: '15.7.0',
```

Comment:
```suggestion
        replacementVersion: '14.0.0',
```
why not v14? it seems it's the oldest available version on npm and we tend to make the smallest step to reduce too many breaking changes.

Reply:
SGTM! 👍